README.md

```diff
@@ -1,31 +1,29 @@
-# Play! 2.3.X plugin for Google Closure Templates
-
-This plugin is designed for using Google Closure Templates with Play! 2.3.X
-
-## Prerequisites
-
-The latest Google Closure Templates jar from google: http://code.google.com/p/closure-templates/
-
-Install the latest sbt-closure-templates plugin: https://github.com/gawkermedia/sbt-closure-templates
+# Play! 2.3.x plugin for Google Closure Templates
 
+This plugin is designed for using Google Closure Templates with Play! 2.3.x
 
 ## Installation
 
-1. in ```project/Build.scala``` add ```"com.kinja.play" %% "play2-closure" % "0.41-2.3.4-SNAPSHOT"``` to your ```project/Build.scala``` file's ```app dependencies``` section.
+1. Clone the repository
```

Comment:
I agree that it could be published but then the interface should be cleaned up, tests and more thorough documentation should be added, etc. So I think this library is just not ready yet to be published to Maven Central. If people want to use it, they can, at their own discretion, without "official" support.